scapy/layers/rtp.py

```diff
@@ -7,8 +7,8 @@
 RTP (Real-time Transport Protocol).
 """
 
-from scapy.fields import *
 from scapy.packet import *
+from scapy.fields import *
```

Comment:
Please remove this changes as it is useless (unless of course if it is useful but I cannot see how).